pages/CitationStyleLanguageHandler.inc.php

```diff
@@ -94,54 +97,45 @@ public function _setupRequest($args, $request) {
 
 		$this->citationStyle = $args[0];
 		$this->returnJson = isset($userVars['return']) && $userVars['return'] === 'json';
-		$this->article = Services::get('submission')->get($userVars['submissionId']);
-
-		if (!$this->article) {
+		$this->submission = Services::get('submission')->get($userVars['submissionId']);
+		$this->issue = $userVars['issueId'] ? Services::get('issue')->get($userVars['issueId']) : null;
+		$this->submissionNoun = $userVars['submissionNoun'];
```

Comment:
Best not to rely on the user request for something like this. We can determine it from the application type.